### ISSUE TITLE
Migration Issue

### ISSUE THREAD

**Issue by bandirom** [mathiasertl/django-ca#79]
Version: django-ca==1.19.0

Issue with migrations

![image](https://user-images.githubusercontent.com/46032838/146154364-d0527d26-b457-467d-ba0a-7d1fc6fb3633.png)

Thanks!

**Comment by mathiasertl** [comment]
Hi,

Can you describe a bit more: what versions of python, django and Django-ca are you using?

**Comment by bandirom** [comment]
Hi!
Thanks for reply!
django=3.2.10
django-ca=1.19.0

Update django_ca from 1.16.0 to 1.19.0
![image](https://user-images.githubusercontent.com/46032838/146507108-ed01d603-0e9e-473b-81b6-f4608c46f56d.png)

There are migrations which should be applied
But when I try to check or create new migrations, I received new migration 27 in library
![image](https://user-images.githubusercontent.com/46032838/146507666-b65346e5-c952-49d4-a157-fe6cc38900fc.png)

**Comment by mathiasertl** [comment]
Hi,
I unfortunately still struggle to reproduce this issue. I do have a good theory by now, though. Maybe you can help me confirm it:

First, What's the value of your `DEFAULT_AUTO_FIELD` setting? I'm guessing it is `django.db.models.AutoField` - no "Big" in there, right?

If I'm right, an you please test two things? 

**First,** in `django_ca/app.py`, set `default_auto_field`, like so:

```diff
--- a/ca/django_ca/apps.py
+++ b/ca/django_ca/apps.py
@@ -21,5 +21,6 @@ from django.utils.translation import gettext_lazy as _
 
 
 class DjangoCAConfig(AppConfig):  # pylint: disable=missing-class-docstring
+    default_auto_field = "django.db.models.BigAutoField"
     name = "django_ca"
     verbose_name = _("Certificate Authority")
```

This should make the issue go away cleanly - if my theory is right.

**Second,** in your settings, try setting `DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"`. This should make the django-ca migration go away as well (but will introduce the same migrations in other apps, if you have any).

Can you confirm both? If yes, I will apply the patch to `app.py`, which should be a permanent fix for this scenario. The fix will be in the soon-to-be-released 1.20.0 (so you could leave your patch in place as a workaround, if that is feaseable to you).

**Comment by bandirom** [comment]
Hi!)
Yes, is it.
Currently `DEFAULT_AUTO_FIELD=django.db.models.AutoField` in settings.py

I will test it today!
Thank you!

**Comment by bandirom** [comment]
Hi!
According your suggest I checked 1st test and

![image](https://user-images.githubusercontent.com/46032838/146647174-33d41c35-28bb-4600-92ff-19362723a0f3.png)

Its work :)

![image](https://user-images.githubusercontent.com/46032838/146647231-8dd47882-ac49-42ce-9d26-09c5bb3ed18e.png)

Thank you!

**Comment by mathiasertl** [comment]
Hi,

I will be releasing django-ca 1.19.1 which will fix the issue (and add explicit support for Django 4.0 and cryptography 36). I will close this issue once it's released (currently still running the tox test suite).

Thank you for using django-ca. Please let me know if you have any further issues!

kr, Mat

**Comment by mathiasertl** [comment]
I just published 1.19.1, it includes the fix from the commit above. Please let me know if you have any further issues.

kr, Mat
